src/test/SDL_test_font.c

```diff
@@ -3126,6 +3126,19 @@ struct SDLTest_CharTextureCache
 */
 static struct SDLTest_CharTextureCache *SDLTest_CharTextureCacheList;
 
+void SDLTest_FreeCaches(void) {
```

Comment:
I suggest calling this SDLTest_FreeTextCache()

Reply:
Thanks.
In the mean time, I found out about the already existing `SDLTest_CleanupTextDrawing`.